### DISCUSSION TITLE
how to remove auth from an application access

### DISCUSSION THREAD

**Discussion by parassjain** [gravitational/teleport#9883] [Q&A] [answered]
does anyone know how to bypass teleport login from a teleport application?
like I don't want to authenticate the user on teleport before using an app

**Comment by zen** [comment, marked as answer]
What you want to achieve is completely against teleport philosophy. Teleport job is to actually authenticate and authorize users. This is what is mainly does. If you don't want to authorize users just put your app in the wild. You can however reuse Teleport auth headers in your app

**Reply by webvictim** [reply]
Yes, 100%. Teleport is not intended to be an open gateway for apps - it requires an authenticated user for anything that accesses resources inside a cluster.

**Reply by parassjain** [reply]
how can I host the application to the public via teleport?
teleport is almost perfect, it sets up everything from SSL, hosting, routing.....
if there is any way to disable auth on a single application(so that it can be public), how can we do that?

**Reply by zen** [reply]
Did you read the thread? Looks like not

**Reply by webvictim** [reply]
@prsjn You can't. Use a regular reverse proxy like Nginx. [Caddy](https://caddyserver.com) also does automatic HTTPS if that's important for you.
